### PR TITLE
mc3 and ccle: work towards styleguide

### PR DESCRIPTION
**Pull request by buchanae**
I wanted to use the MC3 data for some SMMART analysis and I hit some issues.

- adds a missing dependency
- uploaded the MC3 MAF to swift:bmeg/source/mc3
- moved the default options from the readme to the transformer script

I possibly removed too many CLI options. I don't understand how these options get used, such as skip, harvest, no-harvest, filter. Are these for development and debugging?

Still need to make the same fix for CCLE MAF transformer.

### CODE REVIEW

File: transform/mc3/mc3_maf_transform.py [hunk 1]
```diff
@@ -61,10 +63,20 @@ def callset_maker(self, allele, source, centerCol, method, line):
         return sample_calls, sample_callsets
 
 
-def transform(mafpath, prefix, workers=5, skip=0, harvest=True, filter=[]):
-    return parent_transform(mafpath, prefix, workers, skip, harvest, filter,
-                            transformer=MC3_MAFTransformer())
+parser = default_argument_parser()
```
Comment (bwalsh) [inline transform/mc3/mc3_maf_transform.py]: Should this arg parsing logic be wrapped in an `if main ...` ? 
(Causes tests to fail)

File: transform/mc3/mc3_maf_transform.py [hunk 2]
```diff
@@ -61,10 +63,20 @@ def callset_maker(self, allele, source, centerCol, method, line):
...
+options = parser.parse_args()
+default_logging(options.loglevel)
+
+emitter = JSONEmitter("mc3")
 
+maf_path = "source/mc3/mc3.v0.2.8.PUBLIC.maf.gz"
```
Comment (bwalsh) [inline transform/mc3/mc3_maf_transform.py]: this path can't be overwritten by argument?

**Comment by bwalsh** [comment]
I noticed mc3.v0.2.8.PUBLIC.maf.gz here:

Name source/mc3.v0.2.8.PUBLIC.maf.gz
Hash 639ad8f8386e98dacc22e439188aa8fa
Content Type application/octet-stream
Timestamp Apr 9, 2018 6:57:39 PM
Size 718.44 MB

I assumed it was there for a reason?

File: transform/mc3/mc3_maf_transform.py (hunk 1 above)
Comment (buchanae) [reply transform/mc3/mc3_maf_transform.py]: Ah, I didn't realize you were importing the transformer code for testing. Makes sense. I will add that back.

**Comment by buchanae** [comment]
> Name source/mc3.v0.2.8.PUBLIC.maf.gz

Probably just a bad upload. Will fix.

**Pull request closed without merging**